### ISSUE TITLE
Provide clearer accessible labelling for component page navigation landmarks

### ISSUE THREAD

**Issue by DBD324** [mi6/ic-design-system#610]
## Summary

On the component page there are multiple navigation elements. These include:
- Top navigation nav bar
- Left hand navigation
- Page header nav bar
- Anchor nav

When conducted a11y testing with voiceover, these navigations were not announced with a title, making it hard to understand which navigation element you are on. 

For example, when focussed on the left hand nav, you have a list of 34 links, then you move to page header nav bar which is a list of 3 links, then you move to anchor nav which is a list of many links again. 

It is hard to understand which navigation is which and therefore clear labelling of each navigation's purpose should be used. 

## 💬 Description

Ensure that navigation landmarks are labelled clearly and are announced by both NVDA and Voiceover. 

NVDA announces the following for the different navigations, but these could be improved upon:

*Main navigation*
NVDA announcement = 'Banner landmark main navigation navigation landmark'
Voiceover announcement = tbc

Updated announcement = 'Banner landmark main pages navigation landmark'. 

Aria label is currently 'main navigation' but use of word navigation in the label should be replaced as it causes repetition. Instead use 'Main pages' as it is more descriptive. 

'banner landmark' is the separate landmark for the whole banner and is not always read out. 

*Left hand navigation*
NVDA announcement = 'Components section navigation landmark list with 34 items'
Voiceover announcement = tbc

Updated announcement = 'Component pages navigation landmark list with 34 items'

Aria-labelled-by uses 'Components section' but this is not very clear. Instead use 'Component pages'. 

*Page header navigation*
NVDA announcement = 'Navigation landmark page header navigation landmark list with three items'
Voiceover announcement = tbc

Updated announcement = 'Dialog page sections navigation landmark list with three items'

aria-label is currently 'navigation-landmark-page-header'. This is not very clear and can be updated to reference the specific page this appears on. Use '[Page name] page sections' instead. 

*Anchor navigation*
NVDA announcement = 'Components section contents navigation landmark contents'
Voiceover announcement = tbc

Updated announcement = 'Dialog guidance page contents navigation landmark contents'

aria-label is currently 'components section contents', but would be much better if this was related to the specific page. '[Page name] [current tab] page contents'

The second 'Contents' announcement at the end is caused by the anchor nav title. 

## 💰 Use value

This aids the ease of navigation through the site by users of assistive technology. It makes it clearer what each navigation landmark is for and what they should be used for. 

## 📝 Acceptance Criteria

If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil.

_Given_ that a SR user is navigating through landmarks
_When_ each landmark is focussed
_Then_ it's updated labelling should be used as described above. 

## ✏ Designs

If there's a Figma design file (or other mock-up), include it here.

## 🧾 Guidance
If there's written guidance or documentation, include a link to it here.

## Additional info

Tell us anything else useful to help us understand your suggestion.

**Comment by ASM995** [comment]
Page header and top navigation need to be split out in UI repo tickets.
